R/slice-chop.R

```diff
@@ -0,0 +1,89 @@
+#' Chopping
+#'
+#' @description
+#' - `vec_chop()` provides an efficient method to repeatedly slice a vector. It
+#' captures the pattern of `map(indices, vec_slice, x = x)`.
+#'
+#' - `vec_unchop()` combines a list of vectors into a single vector, placing
```

Comment:
I think you should explicitly mention that it's the inverse of `vec_chop()`, and express the relationship clearly with code (e.g. `vec_unchop(vec_chop(x, indices), indices)` equals `x`?)